bin/phpactor.php

```diff
@@ -1,38 +1,37 @@
 <?php
+use Phpactor\Console\Application;
+
+use Symfony\Component\Debug\Debug;
 
 use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
 
-$autoloadFile = __DIR__ . '/../vendor/autoload.php';
-if (!file_exists($autoloadFile)) {
+foreach ([__DIR__ . '/../../autoload.php', __DIR__ . '/../vendor/autoload.php', __DIR__ . '/vendor/autoload.php'] as $file) {
+    if (file_exists($file)) {
+        define('PHPACTOR_COMPOSER_INSTALL', $file);
```

Comment:
This can be defined as a variable no? Why a constant?

Reply:
As I ported the autoload solver from phpunit, and it was using a constant, I thought it would be OK to use one here. But I redefined it as a variable which is ok too.